frontend/src/components/smart/Character.vue

```diff
@@ -670,7 +670,7 @@ export default Vue.extend({
 .characterImg {
   width: 100%;
   object-fit: contain;
-  max-height: 60vh;
+  max-height: 45vh;
```

Comment:
Does it look good on both gen1 and gen2?

Reply:
Yes.